.github/workflows/main.yml

```diff
@@ -1,8 +1,8 @@
 name: docker-wikibase-actions-workflow
-on: 
-  push:
-    branches: 
-      - 'main'
+
+on:
```

Comment:
I think it would be better to have both. Schedule and push / pull request.

Reply:
added build on push to main